setup.py

```diff
@@ -171,5 +185,5 @@
         "Programming Language :: Python :: 3.7",
         "Topic :: Scientific/Engineering :: Artificial Intelligence",
     ],
-    keywords='datasets machine learning datasets metrics',
-)
+    keywords="datasets machine learning datasets metrics",
+)
```

Comment:
`black` misconfiguration?

Reply:
I have the `black` version `20.8b1` installed.

When i `run black --line-length 119 --target-version py36 ./setup.py`
```
All done! ✨ 🍰 ✨
1 file left unchanged.
```

Are the double quotes wrong? If yes I have to investigate why this happens.

Reply:
I have no idea, @lhoestq is the reference here, it's "his" codebase ;)

Reply:
Copy from our conversation on slack:

Yes if it changes quotes to double quotes it means that your black is not configured like the ones used by the other contributors. We try to avoid this since it can lead to conflicts
(this is not a big deal to change the quote in itself, this is more about the risk of future conflicts)
Maybe you can try to revert the quotes to single quote and try to see why black is changing the quotes to fix it

Reply:
I double-checked my `black` configuration even reinstalled it. Still making double-quotes. I also checked if there is an option for `--single-quotes` and found this [github issue](https://github.com/psf/black/issues/594), which seems like they are not even supporting `single-quotes`. 

I am going to fix it by hand but for me, it seems not to be a `black` issue. Would be nice if you could run `black --line-length 119 --target-version py36 ./setup.py`  

e.g. https://github.com/aws/sagemaker-python-sdk/blob/master/setup.py is having double quotes too. 
Could it be possible that no one has ever run `black` for `setpu.py` since it is not in `make style`? 

If not would I adjust the `"` to `'` to be hands.

Reply:
Indeed you're right: it's just that black has never been run on `setup.py`.
Therefore your local installation of black is actually fine.
We can keep the setup.py this way then (i.e. with the " instead of the ')